microsoft-edge/webview2/how-to/print.md

```diff
@@ -671,7 +670,7 @@ async void PrintToPdf(bool enableLandscape)
   }
 
   var savePicker = new Windows.Storage.Pickers.FileSavePicker();
-  savePicker.FileTypeChoices.Add("Pdf Files", new List<string>() { ".pdf" });
+  savePicker.FileTypeChoices.Add("PDF Files", new List<string>() { ".pdf" });
```

Comment:
@monica-ch - tech review this Suggested Change, it's in code

Reply:
This make sense, I'll update in my PR.

Reply:
using "PDF" per conversation